lux/core/series.py

```diff
@@ -55,3 +54,94 @@ def f(*args, **kwargs):
 
         f._get_axis_number = super(LuxSeries, self)._get_axis_number
         return f
+
+    def __repr__(self):
+        from IPython.display import display
+        from IPython.display import clear_output
+        import ipywidgets as widgets
+        from lux.core.frame import LuxDataFrame
+
+        series_repr = super(LuxSeries, self).__repr__()
+        ldf = LuxDataFrame(self)
+
+        try:
+            if ldf._pandas_only:
+                print(series_repr)
```

Comment:
Is this print statement just for bug testing?

Reply:
The print statements here are equivalent to the `display` function in `_repr_html_`, and so it's necessary to show the Series repr.